bleachbit/Unix.py

```diff
@@ -442,13 +438,11 @@ def apt_autoremove():
 def __is_broken_xdg_desktop_application(config, desktop_pathname):
     """Returns boolean whether application deskop entry file is broken"""
     if not config.has_option('Desktop Entry', 'Exec'):
-        print "info: is_broken_xdg_menu: missing required option 'Exec': '%s'" \
-            % (desktop_pathname)
+        logger.info("is_broken_xdg_menu: missing required option 'Exec': '%s'" % desktop_pathname)
```

Comment:
Is `logger` defined here?  See example here for how when I converted another module
https://github.com/az0/bleachbit/commit/6f52a2be7ee73aa3def780133298cdb39ee24a5d